bin/selection_utils.py

```diff
@@ -161,58 +161,98 @@ def target_distance_fcn(args, this_seq, target_seqs):
     return itarget, tdist
 
 # ----------------------------------------------------------------------------------------
+def count_aa(aa_seq,which_aa):
```

Comment:
i could be mistaken, but i think this is equivalent to aa_seq.count(which_aa)?

Reply:
Ah, I missed that function entirely. Where is it located?

Reply:
oh, it's just a standard list function https://docs.python.org/3/tutorial/datastructures.html#more-on-lists